lib/graphql/transaction.ts

```diff
@@ -1,13 +1,18 @@
 import { IterableModel } from "./iterableModel";
 import { ResultPage } from "./resultPage";
 import {
+  AccountTransactionsArgs,
+  BaseOperator,
   MutationCategorizeTransactionArgs,
   Query,
   Transaction as TransactionModel,
+  TransactionFilter,
   TransactionsConnectionEdge,
 } from "./schema";
 import { FetchOptions, Subscription, SubscriptionType } from "./types";
 
+const MAX_SEARCH_QUERY_LENGTH = 200;
```

Comment:
What do you think of truncating user query if too long? is 200 chars proper?

Do you also think I should perform some other sanitization operations before sending the stuff to the backend? 🤔

Reply:
200 I a lot and should be enough, I don't think that user would type that much or paste that long strings

Reply:
200 characters should be enough.